.github/workflows/build-and-test-other.yaml

```diff
@@ -129,7 +129,7 @@ jobs:
         make AtomVM &&
         make test-erlang &&
         file ./tests/test-erlang &&
-        ./tests/test-erlang &&
+        ./tests/test-erlang -s floatmath &&
```

Comment:
`floatmath` test doesn't work with qemu?

Reply:
As mentioned offline by @fadushin this test fails with qemu.

Reply:
It fails on amrel only.  I have adjusted the action to only disable the test on `amrel` arch.